AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -28,6 +28,9 @@ instagram.com##._aagw:empty
 !
 ! End: Instagram.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176470
+venturebeat.com##.post-boilerplate:has(.ICCta)
+venturebeat.com##.post-boilerplate:has(.Boilerplate__newsletter-container)
```

Comment:
Please move to AnnoyancesFilter\Popups\sections\subscriptions_specific.txt (without a comment)